pennylane/devices/experimental/device_api.py

```diff
@@ -203,9 +204,8 @@ def wires(self) -> Wires:
 
     def preprocess(
```

Comment:
Is `preprocess` now the right name for this method?

Before, the verb `preprocess` seemed more fitting since the method would map over input `circuits` and perform the preprocessing. Now, the method gives you the transform program that enacts the preprocessing when applied.

Reply:
What about `preprocess_program`?

Reply:
Decided to think about this at a later date.